kivymd/uix/bottomsheet.py

```diff
@@ -415,14 +415,13 @@ def resize_content_layout(self, content, layout, interval=0):
 Builder.load_string(
     """
 <ListBottomSheetIconLeft>
-    halign: "center"
     theme_text_color: "Primary"
-    valign: "middle"
+
 """
 )
 
 
-class ListBottomSheetIconLeft(ILeftBody, MDIcon):
+class ListBottomSheetIconLeft(ILeftBody, MDIconButton):
     pass
 
 
```

Comment:
Since the class is using mdiconbutton, this properties are not required anymore.